web/src/components/Cart/Cart.js

```diff
@@ -94,12 +95,24 @@ const ShoppingCartButton = styled(Button)`
     `}
 `
 
-const CartItem = ({ image, quantity, name }) => {
+const RemoveCartButton = styled(Button)`
+  background-color: var(--red-7);
+
+  &:hover {
+    background-color: var(--red-6);
+    transition: background-color 200ms;
+  }
```

Comment:
The `Button` component has `&:hover` styles already that brighten the color so we shouldn't need to double up here
```suggestion
```

Here's the styles I'm referring to:

https://github.com/redwoodjs/example-store-stripe/blob/048546b6e3f6bec65998cac575dd3e1bb2e680ae/web/src/components/Button/Button.js#L87-L91

Reply:
Whoops, missed that. Removed the hover color.